rust/lance-index/src/vector/flat/index.rs

```diff
@@ -228,3 +228,81 @@ impl TryFrom<Quantizer> for FlatQuantizer {
         }
     }
 }
+
+#[derive(Debug, Clone, DeepSizeOf)]
+pub struct FlatBinQuantizer {
```

Comment:
We have to add this rather than reusing `FlatQuantizer` even flat quantizer does nothing, because we determine the `VectorStore` type by `Quantization`